docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -822,6 +828,40 @@
                 </tr>
 
                 <tr>
+                    <th colspan="3" class="section" id="streams"><a href="../container/streams.html">Streams</a></th>
+                </tr>
+
+                <tr>
+                    <td class="property" id="streams-system">streams.<span class="stream">stream-id</span>.samza.system</td>
+                    <td class="default"></td>
+                    <td class="description">
+                        The <span class="system">system-name</span> of the system on which this stream will be accessed.
+                        This property binds the stream to one of the systems defined with the property
+                        systems.<span class="system">system-name</span>.samza.factory. <br>
+                        If this property isn't specified, it is inherited from job.default.system.
+                    </td>
+                </tr>
+
+                <tr>
+                    <td class="property" id="streams-physical-name">streams.<span class="stream">stream-id</span>.samza.physical.name</td>
```

Comment:
@nickpan47 
> we have seen this when one user sending output to local.PageViewEventByMemberId fail to deploy to production since local cluster is not there and the input has to be queuing.PageViewEventByMemberId, even in the code)

I don't agree to this use-case to support your point on code portability. The user can always define app-level configs to indicate inputsystem = "queuing" and outputsystem = "local". They can refer to this in their code without requiring a re-compilation of code. It is just a choice by the user to hard-code the system and/or stream name in code. 

> Co-existence of systems.<xyz>.streams.<abc>. and streams.<abcId>.system... is confusing

+1 on this. I still have question unanswered about whether we are restricting system-scoped properties in the new model. in case @jmakes is OOO, you can address my questions above?